lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -50,10 +52,8 @@ class _CfdOrderDetailState extends State<CfdOrderDetail> {
     formatter.minimumFractionDigits = 2;
     formatter.maximumFractionDigits = 2;
 
-    final cfdTradingService = context.read<CfdTradingChangeNotifier>();
-
     final cfdOffersChangeNotifier = context.watch<CfdOfferChangeNotifier>();
```

Comment:
🙃 I guess we should align the variable name with the type.


```suggestion
    final cfdOfferChangeNotifier = context.watch<CfdOfferChangeNotifier>();
```

Reply:
happy to do that in a follow-up, this leaks through many files and is unrelated to this PR